ddocs/medic/_attachments/translations/messages-en.properties

```diff
@@ -1183,3 +1183,8 @@ branding.favicon.field = Favicon
 branding.title.field = Title
 messages.errors.invalid = Invalid message:
 messages.errors.patient.missing = Patient not found
+partner.tab.partners = Partners
+partner.name.field = Partner Name
+partner.logo.field = Partner Logo
+partner.logo.upload = Upload Partner Logo
+partner.supporting = Supporting Partners
```

Comment:
We use sentence case not title case throughout the app, ie: all letters lowercase except for the start of the sentence. Please update these four translations.